MVP Module/Source/Feed Presentation/FeedPresenter.swift

```
@@ -16,10 +20,12 @@ protocol FeedView {
 final class FeedPresenter {
 	private let feedView: FeedView
 	private let loadingView: FeedLoadingView
+	private let errorView: FeedErrorView
 	
-	init(feedView: FeedView, loadingView: FeedLoadingView) {
+	init(feedView: FeedView, loadingView: FeedLoadingView, errorView: FeedErrorView) {
 		self.feedView = feedView
 		self.loadingView = loadingView
+		self.errorView = errorView
 	}
 	
 	func didStartLoadingFeed() {
```

The Presenter decides when to show the error message, so it should also decide when to hide the message.

This way, the ViewController doesn't need to decide when to hide the error. It's easier to manage state when all changes are centralized in a single component.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `errorView.display(FeedErrorViewModel(errorMessage: .none))` to hide the error.